boards/phpbb3.php

```diff
@@ -8,83 +8,84 @@
  */
```

Comment:
I assume the large diff here is due to some automatic formatting from an IDE/editor?

Reply:
Yeah, recently started using PHPStorm.